api/queries_pr.go

```diff
@@ -584,6 +585,7 @@ func PullRequestByNumber(client *Client, repo ghrepo.Interface, number int) (*Pu
 				milestone{
 					title
 				}
+				viewerMergeBodyText
```

Comment:
This query looks good for github.com, but this field is not available in GitHub Enterprise 2.22 or lower, and we support those as well. Adding this field would break all pull request queries on GHE.

We have a mechanism for detecting whether a GraphQL field is supported on the remote. I will take over this PR from here and make the query safe for different hosts. In the meantime I will mark the PR as "changes requested" to block merging.

Thank you for all your contribution so far!